superset-frontend/src/cccs-viz/plugins/plugin-chart-cccs-grid/src/plugin/transformProps.ts

```diff
@@ -136,6 +138,23 @@ export default function transformProps(chartProps: CccsGridChartProps) {
   },
   sortingColumnMap);
 
+  function compareIPv4(valueA: string, valueB: string, nodeA: RowNode, nodeB: RowNode, isInverted: boolean) {
+    // https://stackoverflow.com/questions/48618635/require-sorting-on-ip-address-using-js
```

Comment:
You don't need to include a reference to where you got the code from since this is going to be internal. Not too sure what the upstream rules on that are though.
```suggestion

```